client/src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import Login from './views/Login';
+
+function App() {
+  return (
+    <>
```

Comment:
why these?

Reply:
I put `<> </>` tags when I create a React component so that I can add adjacent siblings easily in future if needed. These tags also won't add any extra HTML element.